setup/tinyHIPPOstart

```diff
@@ -28,8 +28,9 @@ uci set luci.ccache.enable=0
 uci commit luci
 /etc/init.d/uhttpd restart
 
-# Create the sqlite database
+# Create and populate the sqlite database
 sqlite3 $TINYHIPPO_INSTALL_PATH/tinyHIPPO.db < $TINYHIPPO_INSTALL_PATH/tinyHIPPO_db_setup.sql
+sqlite3 $TINYHIPPO_INSTALL_PATH/tinyHIPPO.db < $TINYHIPPO_INSTALL_PATH/tinyHIPPO_populate_equations.sql
```

Comment:
Will this line properly insert the equations into the SQL database? I haven't played around too much with this syntax so I want to verify.

Reply:
That's what it's supposed to do, if I wrote the SQL insert correctly. ID auto-increments now so I need to remove that field. But the syntax works

Reply:
The `sqlite3` command works kinda like the built in python interpreter, and when you feed it a text input it will run those commands/queries.

Reply:
Perfect. Exactly what we need here